Jenkinsfile_updatecli

```diff
@@ -0,0 +1,4 @@
+updatecli(action: 'diff')
+if (env.BRANCH_IS_PRIMARY) {
+    updatecli(action: 'apply', cronTriggerExpression: '@daily')
+}
```

Comment:
```suggestion
properties([pipelineTriggers([cron('@daily')])])

updatecli(action: 'diff')
if (env.BRANCH_IS_PRIMARY) {
    updatecli(action: 'apply')
}
```

nitpick: Good practice to avoid setting cron triggers inside pipeline libraries